P32-P35_P72-P75.s

```diff
@@ -34,7 +34,7 @@
 #
```

Comment:
Non-spotted typos. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0618.jpg)

`WHITE, P.` should be `WHITE.P`
Date section should be `1JUNE67`